src/test/regress/sql/multi_subquery_in_where_reference_clause.sql

```diff
@@ -0,0 +1,130 @@
+--
+-- queries to test the subquery pushdown on reference tables
+
+-- subqueries in WHERE with greater operator
+SELECT 
+  user_id
+FROM 
+  users_table
+WHERE 
+  value_2 >  
+          (SELECT 
+              max(value_2) 
+           FROM 
+              events_reference_table  
+           WHERE 
+              users_table.user_id = events_reference_table.user_id AND event_type = 50
```

Comment:
I used users_table to test having reference table only in the inner part of the semi join.